pkg/pipelines/bootstrap.go

```diff
@@ -554,37 +508,22 @@ func createCICDResources(fs afero.Fs, repo scm.Repository, pipelineConfig *confi
 	// value: YAML content of the resource
 	outputs := map[string]interface{}{}
 	otherOutputs := map[string]interface{}{}
-	if !o.Insecure {
-		githubSecret, err := secrets.CreateSealedSecret(meta.NamespacedName(cicdNamespace, eventlisteners.GitOpsWebhookSecret),
-			o.SealedSecretsService, o.GitOpsWebhookSecret, eventlisteners.WebhookSecretKey)
-		if err != nil {
-			return nil, nil, fmt.Errorf("failed to generate GitHub Webhook Secret: %w", err)
-		}
-		outputs[secretsPath] = githubSecret
-	} else {
-		githubSecret, err := secrets.CreateUnsealedSecret(meta.NamespacedName(cicdNamespace, eventlisteners.GitOpsWebhookSecret),
-			o.SealedSecretsService, o.GitOpsWebhookSecret, eventlisteners.WebhookSecretKey)
-		if err != nil {
-			return nil, nil, fmt.Errorf("failed to generate GitHub Webhook Unsealed Secret: %w", err)
-		}
-		unEncSecretPath := filepath.Join("secrets", "gitops-webhook-secret.yaml")
-		otherOutputs[unEncSecretPath] = githubSecret
+	githubSecret, err := secrets.CreateUnsealedSecret(meta.NamespacedName(cicdNamespace, eventlisteners.GitOpsWebhookSecret), o.GitOpsWebhookSecret, eventlisteners.WebhookSecretKey)
+	if err != nil {
+		return nil, nil, fmt.Errorf("failed to generate GitHub Webhook Unsealed Secret: %w", err)
```

Comment:
Nit: should we from now on not qualify Secret with unencrypted or unsealed?  Maybe just say '....GitHub Webhook Secret'